src/clr-addons/paged-search-result-list/paged-search-result-list.html

```diff
@@ -1,23 +1,44 @@
-<div class="clr-row">
-  <clr-pager
-    class="pager-top"
-    *ngIf="clrPagerPosition === 'top' || clrPagerPosition === 'both'"
-    [clrPage]="currentPage"
-    [clrPageSize]="pageSize"
-    [clrTotalItems]="totalItems"
-    (clrPageChange)="pageChange.emit($event)"
-  ></clr-pager>
+<div *ngIf="clrPagerPosition === 'top' || clrPagerPosition === 'both'">
+  <ng-container *ngTemplateOutlet="pager"></ng-container>
 </div>
 <ng-container *ngFor="let item of items" [ngTemplateOutlet]="itemTemplate" [ngTemplateOutletContext]="{item: item}">
   <div *ngIf="!itemTemplate">Item template input is undefined!</div>
 </ng-container>
-<div class="clr-row">
-  <clr-pager
-    class="pager-bottom"
-    *ngIf="clrPagerPosition === 'bottom' || clrPagerPosition === 'both'"
-    [clrPage]="currentPage"
-    [clrPageSize]="pageSize"
-    [clrTotalItems]="totalItems"
-    (clrPageChange)="pageChange.emit($event)"
-  ></clr-pager>
+<div *ngIf="clrPagerPosition === 'bottom' || clrPagerPosition === 'both'">
+  <ng-container *ngTemplateOutlet="pager"></ng-container>
 </div>
+<ng-template #pager>
+  <div class="clr-row">
+    <div
+      [class.pager-top]="clrPagerPosition === 'top' || clrPagerPosition === 'both'"
+      [class.pager-bottom]="clrPagerPosition === 'bottom' || clrPagerPosition === 'both'"
```

Comment:
pager-top and pager-bottom should not depend on the provided options, but on the location where the pager is shown.
because now if you set 'both' the pager at the top also gets the class 'pager-bottom', right?